### PR TITLE
#1675 waived interview other info fields

### PR DESCRIPTION
**Pull request by courtrightd**
Adds two fields to the information needed dialog that allow the user to enter additional information that comes up in the conversation.
These fields are added to the information provided case note. If left blank, nothing is added to note.

### CODE REVIEW

File: notes/snap-waived-interview.vbs [hunk 1]
```diff
@@ -271,7 +288,7 @@ Function needed_info_dialog(needed_info_array)
 		If previous_verif_found = True Then PushButton 20, 350, 125, 15, "View Previous Verifications", view_previous_verifs_btn
 	End If
 	PushButton 345, 350, 75, 15, "Complete Contact", contact_completed
-	If last_page <> page_display Then PushButton 290, 350, 50, 15, "Next", next_btn
+	If page_display <> q_page_xtra Then PushButton 290, 350, 50, 15, "Next", next_btn
```
Comment (MeganGeissler) [inline notes/snap-waived-interview.vbs:291]: Verification buttons on the xtra dialog do not provide a dialog to enter information. Seems like it's looping back to the xtra dialog without providing verf dialog.

**Review by MeganGeissler — CHANGES_REQUESTED** [review]
Added a few comments to review! Otherwise the casenote looks good.

**Review by IlseFerris — CHANGES_REQUESTED** [review]
The dimming worked for the contact information. 
I was still unable to get the other information verifications to pull through.

**Comment by courtrightd** [comment]
Updates made that seem to have fixed the verifications pulling through for me....

**Review by C-Love — APPROVED** [review]

**Comment by C-Love** [comment]
I've reviewed the code and it all appears solid. I have not had time to run the script as a test. 

We have a separate syntax bug reported, so I have added this fix to the branch and I am pulling it all in together.

**Pull request merged**